junit-jupiter-migrationsupport/src/main/java/org/junit/jupiter/migrationsupport/rules/TestRuleSupport.java

```diff
@@ -93,10 +94,11 @@
 	}
 
 	@Override
-	public void beforeEach(ExtensionContext context) throws Exception {
+	public void beforeEach(ExtensionContext context) {
 		invokeAppropriateMethodOnRuleAnnotatedMembers(context, NO_OP, GenericBeforeAndAfterAdvice::before);
 	}
 
+	@SuppressWarnings("ConstantConditions")
```

Comment:
Is this one for `ExceptionUtils.throwAsUncheckedException`?

Reply:
Yes. The compiler complains of `ExceptionUtils.throwAsUncheckedException` may produce NPE but the method will never return null